core/modules/replica/ControllerApp.h

```diff
@@ -111,6 +112,24 @@ class ControllerApp : public Application {
     /// A database password for establishing a connection with the worker's database
     std::string _sqlPassword;
 
+    /// The name of a database 
+    std::string _sqlDatabase;
+
+    /// The name of a table 
+    std::string _sqlTable;
+
+    /// The name of a MySQL engine for tables to be created
+    std::string _sqlEngine;
+
+    /// The name of a file where to read table schema from
+    std::string _sqlSchemaFile;
+
+    /// The name of the PK for the MySQL partitioned tables
```

Comment:
what's a PK?

Reply:
The PRIMARY KEY of a table. I will correct the comment